buildout.cfg

```diff
@@ -18,6 +18,9 @@ parts +=
     rebuild_i18n-sh
     robot
 
+test-eggs +=
+    plone4.csrffixes
+
```

Comment:
@hvelarde should I run this just in 4.3 version? If so, I need to figure out when CSRF is enabled into RF test.